src/JuMP.jl

```diff
@@ -738,5 +729,26 @@ include("print.jl")
 # Nonlinear-specific code
 include("nlp.jl")
 
+
+typealias _getTypes Union{NonlinearExpression,NonlinearParameter, AffExpr}
```

Comment:
Actually, it was because the `getValue(Array{Variable})` function seems to be doing a lot more checking and Warning printing than was done on the original `getValue(Array{AffExpr})`. 

I didn't want to remove all that from `getValue(Array{Variable})`